monitor_node_activity.py

```
@@ -0,0 +1,203 @@
+def log_message(message):
+    logging.debug(message)
+    print(message)
+
+
+def update_blocked_node(node_id, comment=""):
+    sanitized_comment = comment.replace(" ", "_")
+    command = '{} {} -rx "database put {} {} {}"'.format(
+        SUDO, ASTERISK, DB_FAMILY, node_id, sanitized_comment
+    )
+    log_message("Running command: {}".format(command))
+    subprocess.run(command, shell=True, check=True)
```

Ruff (0.14.13)

`subprocess` call with `shell=True` identified, security issue

(S602)